index.html

```diff
@@ -90,7 +83,8 @@ <h4 class="h-4">Tonic</h4>
                 <img class="work-pic" src="./images/desktop-images/img1.png" alt="landing page view" />
                 <img class="work-pic pop-img" src="./images/Portoflio_Card/work.png" alt="nature image" />
                 <div class="pop-element">
-                    <p class="description-text descriptionTxt">
+                    <p class="description-text">
+                        Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
```

Comment:
https://github.com/Epaltechs/Setup-Portfolio/blob/ba35ef19bbc9b95404196a900ad8e753f93c33ab/index.html#L66-L231

- [ ] In this particular project, the whole projects section along with the modals should be created dynamically with Javascript.

The base projects section is already done (good job!), but the modals are still not being created with Javascript.  Kindly please use Javascript to dynamically render your modals.

You could reuse the code that you already have.

That way, adding projects in the future will be easy. You will only need to add it in one place - as another object in the `data` array.

![image](https://user-images.githubusercontent.com/57421823/152509546-f41b40b6-dc88-4e3f-9ab4-7bee886e1f52.png)

- Also, you could put some id on all the projects, and make sure the id of each project matches the id of the said project on the data array. Then, iteratively add an event listener to each button calling the correct id, so that the correct project id is shown when pressed. (These are just some ideas I hope it's helpful).

- [ ] It would be better for the background not to be scrollable when the popup windows is open . you can use the CSS property overflow: hidden; to prevent that.